src/Compilers/Core/MSBuildTask/Microsoft.Managed.Core.targets

```diff
@@ -64,6 +64,80 @@
     <EditorConfigFiles Include="@(PotentialEditorConfigFiles->Exists())" Condition="'$(DiscoverEditorConfigFiles)' != 'false'" />
   </ItemGroup>
 
+  <!--
```

Comment:
> <!-- [](start = 1, length = 5)

nit: formatting

Reply:
Tested with a test project, lots of combinations of values and the msbuild log viewer. Basically copy the updated targets into a directory.build.targets, and change the task assembly path to the Roslyn source version. We should definitely think about investing in some sort of structured tests for these files.

---
In reply to: [419698161](https://github.com/dotnet/roslyn/pull/43617#discussion_r419698161) [](ancestors = 419698161)